README.md

```diff
@@ -1,9 +1,12 @@
 # TiFlow
 
+[![LICENSE](https://img.shields.io/github/license/pingcap/tiflow.svg)](https://github.com/pingcap/tiflow/blob/master/LICENSE)
```

Comment:
see a lot of changes about readme and cdc, is that related to fixing this bug?

Reply:
She possibly rebase-merged the master🤔

Reply:
Right. I rebase-merged the master into my branch, probably shouldn't have done so. Only the changes in (*SubTask).Result() in dm/dm/worker/subtask.go and TestSubtaskRace() in dm/dm/worker/subtask_test.go are related to fixing this bug. I have fixed this problem, and now this PR should only contain the changes above.